source/shared/cpp/ObjectModel/ActionParserRegistration.h

```diff
@@ -14,14 +14,32 @@ namespace AdaptiveSharedNamespace
         virtual std::shared_ptr<BaseActionElement> Deserialize(ParseContext& context, const Json::Value& value) = 0;
     };
 
+    class ActionElementParserWrapper : public ActionElementParser
```

Comment:
Lets add a comment in here explaining why we need the wrapper

Reply:
Done!

Reply:
I see that the mega comment covers this, but maybe reference it here ("See ParseContext.cpp for details")

---
In reply to: [255195775](https://github.com/Microsoft/AdaptiveCards/pull/2351#discussion_r255195775) [](ancestors = 255195775)

Reply:
Ah, nevermind, i see the comment where it's used which is probably just as good.

---
In reply to: [257416041](https://github.com/Microsoft/AdaptiveCards/pull/2351#discussion_r257416041) [](ancestors = 257416041,255195775)